openverse_catalog/dags/providers/provider_api_scripts/freesound.py

```diff
@@ -42,13 +43,13 @@ class FreesoundDataIngester(ProviderDataIngester):
         "preview-lq-ogg": 80000,
     }
 
-    headers = {
-        "Accept": "application/json",
-    }
-
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.api_key = Variable.get("API_KEY_FREESOUND")
+        self.headers = {
+            "Accept": "application/json",
+            "Authorization": f"Token {self.api_key}",
+        }
```

Comment:
Doesn't the call to the parent constructor has to be after setting the headers for `DelayedRequester` to be configured correctly?

Reply:
😮 Good point! This currently works as is because `get_response_json` also explicitly passes `self.headers` in [here](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py#L353). But it would certainly cause a problem if we ever used `self.delayed_requester` directly in the child class, which we actually have plans to do in https://github.com/WordPress/openverse-catalog/issues/754! And it's good practice, besides.

Done, thanks 😄

Reply:
Perfect!